myeongseok/app.js

```diff
@@ -35,23 +37,58 @@ app.get('/ping', (req, res, next) => {
   res.status(200).json({ message: 'pong' });
 });
 
-app.post('/users/signup', (req, res, next) => {
+app.post('/users/signup', async (req, res, next) => {
   const { name, email, profileImage, password } = req.body;
+  const saltRounds = 12;
```

Comment:
```suggestion
  const saltRounds = process.env. ...;
```
보안과 관련된 부분이니 환경 변수로 따로 관리해줘도 좋을 것 같아요~

Reply:
네 감사합니다!